_spinnaker/policy_engine.md

```diff
@@ -16,7 +16,7 @@ The Policy Engine has been tested with OPA versions 0.12.x and 0.13.x
 
 ## Before You Start
 Keep the following guidelines in mind when using the Policy Engine: 
-* The Policy Engine uses 'fail closed' behavior. That means that if you have the policy engine enabled but no policies created, Spinnaker refuses to create or update any pipeline. 
+* The Policy Engine uses **fail closed** behavior. That means that if you have the policy engine enabled but no policies created, Spinnaker refuses to create or update any pipeline. 
```

Comment:
from my testing - or the way how I configured OPA possibly - this wasn't true.

Reply:
Eeek. We should probably get a confirmation about this behavior. I pinged the product channel for policy engine.

Reply:
looks like i did not fully test this.